modules/services/imapnotify-accounts.nix

```diff
@@ -30,10 +30,15 @@ with lib;
       description = "IMAP folders to watch.";
     };
 
+    extraArgs = mkOption {
+      type = types.listOf types.str;
+      default = [ "-wait 1" ];
```

Comment:
the current module doesn't seem to set `wait` at all. Can we keep this behavior and use "-wait 1" as an example instead ?